main.py

```diff
@@ -87,10 +87,23 @@ def train(chunk_dir, save_file, load_file=None, generation_num=0,
 
     print("Training from:", tf_records[0], "to", tf_records[-1])
 
-    n = dual_net.DualNetworkTrainer(save_file)
+    n = dual_net.DualNetworkTrainer(save_file, logdir=logdir)
     with timer("Training"):
         n.train(tf_records, init_from=load_file,
-                logdir=logdir, num_steps=num_steps, verbosity=verbosity)
+                num_steps=num_steps, verbosity=verbosity)
+
+
+def validate(tf_records, load_file=None, logdir=None, num_steps=1000):
```

Comment:
I'm not sure I agree with passing in tf_records via command line args. It will almost certainly run into restrictions for bash wildcard expansion or process limits on sys.argv or something weird like that.

Reply:
Also I think for this to work properly via command line, arghparse expects the signature to be `*tf_records`

Anyway, what about passing in *tf_record_dirs, and then globbing inside main.validate() to recursively find all contained tf_record files?

Reply:
oh good point.  What would you say to making validate *not* a CLI command, only an rl_loop command?

Reply:
main.validate now takes a list of directories ;)  Done & merging!